doc/newfeat.texi

```diff
@@ -107,15 +107,22 @@ will need to update their customization to
 @code{inferior-ess-R-program}. These variables are treated as risky
 variables.
 
-@item Customization of ess-smart-S-assign-key has been reworked.
-Use @code{(setq ess-smart-S-assign-key nil)} to disable smart
-assignment at any time instead of @code{(ess-toggle-underscore nil)}.
-To use another key, you should set the value of
-@code{ess-smart-S-assign-key} before ESS is loaded. The following
-functions have been made obsolete. You should customize
-ess-smart-S-assign-key instead: ess-toggle-S-assign,
-ess-toggle-S-assign-key, ess--unset-smart-S-assign-key,
-ess--activate-smart-S-assign-key, ess-disable-smart-S-assign
+@item @code{ess-smart-S-assign} changed name to @code{ess-insert-S-assign}.
```

Comment:
How about @lionel-'s earlier comment on renaming this one. 

Should we maybe name it `ess-insert<-` or `ess-smart<-`?

Reply:
Sure, easy enough to do. I don't like using `<-` in the name since users can customize what assignment operator gets used (and non-R languages could use something else). 

How about `ess-insert-assign`?

Reply:
> How about ess-insert-assign?

Sounds good.

Reply:
OK, done